### PR TITLE
fix:resolved sign-up distortion[#2133]

### PR DESCRIPTION
**Pull request by aakashreddy-p**
Updated the CSS properties so that it can take large emails[#2133].

### CODE REVIEW

File: src/pages/Auth/verify-email/VerifyEmail.svelte [hunk 1]
```diff
@@ -243,11 +243,10 @@
...
-
-				<div class="sparrow-fs-14 sparrow-fs-300">
+				<div class="d-flex flex-column mx-auto sparrow-fs-14 sparrow-fs-300">
 					<p>
-						Check your inbox for the verification code sent to <span
-							class="sparrow-fw-700 text-whiteColor cursor-pointer">{emailText}</span
+						Check your inbox for the verification code sent to<span
+							class="sparrow-fw-700 text-whiteColor cursor-pointer email-text d-inline-block">{emailText}</span
 						> and enter it below to proceed.
 					</p>
```
Comment (itsmdasifraza) [inline src/pages/Auth/verify-email/VerifyEmail.svelte:251]: add the limit to crop text, if it  overflows flows

**Comment by itsmdasifraza** [comment]
Could you change the base branch to development

**Review by Astitva877 — CHANGES_REQUESTED** [review]
@aakashreddy-p whenever you raise PR, raise it in development not in main.

File: src/pages/Auth/verify-email/VerifyEmail.svelte (hunk 1 above)
Comment (aakashreddy-p) [reply src/pages/Auth/verify-email/VerifyEmail.svelte]: @itsmdasifraza I will fix it.

File: src/pages/Auth/verify-email/VerifyEmail.svelte [hunk 2]
```diff
@@ -21,6 +21,7 @@
 	const verifyString = writable('');
 	let verifyLength: string = '';
 	let isRegistered = false;
+	let maxlength = 40;
 	let redirectRules = {
```
Comment (itsmdasifraza) [inline src/pages/Auth/verify-email/VerifyEmail.svelte:25]: use 
```
   const  MAX_LENGTH = 100;
```

File: src/pages/Auth/verify-email/VerifyEmail.svelte [hunk 3]
```diff
@@ -223,6 +224,14 @@
 		onCodeInput();
 		handleVerificationCode();
 	}
+	// This function will reduce the email size if it is too long.
+	function handleEmail(text:string,maxlength:number){
+		if(text.length > maxlength){
+			return text.slice(0, maxlength) + "..."
+		}
```
Comment (itsmdasifraza) [inline src/pages/Auth/verify-email/VerifyEmail.svelte:231]: use arrow function here, return type is missing

**Review by itsmdasifraza — CHANGES_REQUESTED** [review]

File: src/pages/Auth/verify-email/VerifyEmail.svelte (hunk 2 above)
Comment (aakashreddy-p) [reply src/pages/Auth/verify-email/VerifyEmail.svelte]: okay

File: src/pages/Auth/verify-email/VerifyEmail.svelte (hunk 3 above)
Comment (aakashreddy-p) [reply src/pages/Auth/verify-email/VerifyEmail.svelte]: I will fix it

File: src/pages/Auth/verify-email/VerifyEmail.svelte [hunk 4]
```diff
@@ -223,6 +224,14 @@
...
+		return text
+	}
+	$: EmailText = handleEmail(emailText, maxlength);
```
[the remark this answers is not in the document]
Comment (aakashreddy-p) [reply src/pages/Auth/verify-email/VerifyEmail.svelte]: I will change it.

**Pull request closed without merging**